smartmodules/jolt/hub/package-meta.yaml

```diff
@@ -0,0 +1,10 @@
+---
+package_format_version: "0.1"
+name: jolt
+version: 0.1.0
```

Comment:
I think this file is not needed. It should be automatically generated by hub publish. 

Lets have separate issue to have release CI job for publishing

Reply:
It is generated with paths to the crate's `target` folder, but in our case, we need the workspace's `target` folder. So we need to manually modify this file.

Reply:
Got it. @superstator, need to fix this https://github.com/infinyon/fluvio/issues/2652